.github/workflows/reusable-build-test-release.yml

```diff
@@ -799,6 +816,38 @@ jobs:
           echo "spl-host-suffix=wfe.splgdi.com"
           echo "k8s-manifests-branch=main"
           } >> "$GITHUB_OUTPUT"
+      - uses: actions/download-artifact@v3
+        if: ${{ needs.test-inventory.outputs.ucc_modinput_functional == 'true' }}
+        id: download-openapi
+        with:
+          name: artifact-openapi
+          path: ${{ github.workspace }}
+      - name: Setup python
+        if: steps.download-openapi.conclusion != 'skipped'
+        uses: actions/setup-python@v4
+        with:
+          python-version: 3.7
+      - name: modinput-test-prerequisites
+        if: steps.download-openapi.conclusion != 'skipped'
+        shell: bash
+        env:
+          PYTHON_KEYRING_BACKEND: keyring.backends.null.Keyring
```

Comment:
What is the purpose of this?

Reply:
Yet another poetry bug: https://github.com/python-poetry/poetry/issues/1917

I will add this to PR description.